client/src/pages/Profile/Profile.tsx

```diff
@@ -21,7 +25,7 @@ export default function Profile(): JSX.Element {
                         <List component="nav" className={classes.menuWrapper}>
                             {listText.map((text, index) => {
                                 return(
-                                <ListItem key={index} button>
+                                <ListItem selected={index === compIndex} key={index} button onClick={() => setIndex(index)}>
```

Comment:
This solution is functional but if possible, look into refactoring the code to use 'react-router' dom links here (that way we can have individual routes for each of the pages within settings)